sdk/storage/storage-queue/test/aborter.spec.ts

```diff
@@ -39,7 +39,7 @@ describe("Aborter", () => {
       await response;
       assert.fail();
     } catch (err) {
-      assert.equal(err.message, "The request was aborted", "Unexpected error caught: " + err);
+      assert.equal(err.message, "The operation was aborted.", "Unexpected error caught: " + err);
```

Comment:
We should check for `error.name` as well?

Reply:
Sure, will update. The find/replace of the err message text uncovered more places to add that check.

Reply:
Updated the storage trifecta tests.